### PR TITLE
Added endpoint for adding identifier

### PR DESCRIPTION
**Pull request by rogers89**

### CODE REVIEW

File: glottolog3/__init__.py [hunk 1]
```diff
@@ -137,6 +137,14 @@ def main(global_config, **settings):
         views.languages,
         renderer='language/bpsearch_html.mako')
 
+    config.add_route_and_view(
+        'glottolog.add_identifier', 
+        '/resource/identifier/add',
```
Comment (baieric) [inline glottolog3/__init__.py]: Let's use RESTful API design. This endpoint should be POST request to `/identifiers`. I believe you can also use `config.add_route('function_name', '/identifiers')`, which will just use function_name, which needs a `@view_config` modifier. You can search up this modifier to see how it's used in glottolog, or check out this resource: https://zaiste.net/building_a_restful_api_with_pyramid_resource_and_traversal/

**Review by baieric — CHANGES_REQUESTED** [review]
let's remember to add unit tests once we have validation checks. Besides that, let's just change this to RESTful design.

**Review by baieric — APPROVED** [review]
LGTM!

**Comment by ioanacrant** [comment]
Generally, I believe RESTful design says that the response for a POST request should be the JSON representation of the object just added and letting the status code (200) be the check that the call succeeded

File: glottolog3/views.py [hunk 2]
```diff
@@ -318,6 +321,47 @@ def bpsearch(request):
...
+        lang=lang)
+
+    try:
+        DBSession.add(identifier)
+        DBSession.add(
+            LanguageIdentifier(language=languoid, identifier=identifier))
+        DBSession.flush()
+    except exc.SQLAlchemyError as e:
+        DBSession.rollback()
+        return {'message': 'ERROR: {}'.format(e)}
```
Comment (baieric) [inline glottolog3/views.py]: Ioana uses some good conventions in #12, let's change this to just be `{ 'error': '{}'.format(e) }`

**Comment by rogers89** [comment]
Currently it just returns the name of the identifier because the SQL representation in JSON is really ugly. We can change when we add a marshmallow schema for identifiers.

**Pull request merged**